packages/relay-signer/src/models/relayer.ts

```diff
@@ -53,7 +54,11 @@ export interface UpdateRelayerPoliciesRequest {
   gasPriceCap?: BigUInt;
   whitelistReceivers?: Address[];
   EIP1559Pricing?: boolean;
-  privateTransactions?: boolean;
+  /**
+   * Allowing boolean for backwards compatibility.
+   * New relayers should use PrivateTransactionMode.
+   */
+  privateTransactions?: boolean | PrivateTransactionMode;
```

Comment:
Why not using a new param here? like `privateTransactionsMode`

Reply:
I was thinking that originally, but it's part of the same policy. Adding a new parameter might get misinterpreted as a new policy.